qbft/spectest/tests/happy_flow.go

```diff
@@ -32,3 +35,36 @@ func HappyFlow() SpecTest {
 		},
 	}
 }
+
+func happyFlowStateComparison() *qbftcomparable.StateComparison {
```

Comment:
just for future safety I would accept `ks`, and `state` to this function, just because if someone decides to change them in `HappyFlow` then they won't match.
running this function on the already initialized key shares and state will fool-proof it.

Reply:
noted. this design suggested by Alon.
@GalRogozinski pls approve if we want to adjust this logic

Reply:
I think it is fine the way it is now.
The idea here is to have a seperation between the expected state and the test logic.

I think it is better for the dev to actively change the state as needed.